pkg/spec/gitops.go

```diff
@@ -105,9 +106,9 @@ func parseName(raw json.RawMessage, result *GitOps, multiError *multierror.Error
 	if result.TeamName == nil || *result.TeamName == "" {
 		return multierror.Append(multiError, errors.New("team 'name' is required"))
 	}
-	if !isASCII(*result.TeamName) {
-		multiError = multierror.Append(multiError, fmt.Errorf("team name must be in ASCII: %s", *result.TeamName))
-	}
+	// Normalize team name for full Unicode support, so that we can assume team names are unique going forward
+	normalized := norm.NFC.String(*result.TeamName)
```

Comment:
Is this golang normalization something that is susceptible to changes in the future?

If so, will this code support stored names that were normalized with a previous version of normalizer?
If not, --> all good. Ignore.

Reply:
Good question. Normalization is backward compatible. From https://unicode.org/reports/tr15/:
> the results of normalizing a string on one version will always be the same as normalizing it on any other version

The version of Unicode used matches that of the compiler, which is 15.0 for Go 1.21 and 1.22

Reply:
Sounds good then !

Reply:
@sharon-fdm Can you approve?